sklearn/model_selection/_search.py

```diff
@@ -1201,7 +1211,7 @@ class RandomizedSearchCV(BaseSearchCV):
         step, which will always raise the error.
 
     return_train_score : boolean, default=True
-        If ``'False'``, the ``cv_results_`` attribute will not include training
+        If ``False``, the ``cv_results_`` attribute will not include training
```

Comment:
Why does this differ from in GridSearchCV?

Reply:
What do I need to do here?

Reply:
Make this match the parallel comment in GridSearchCV

Reply:
Make this match the parallel comment in GridSearchCV

Reply:
But why should it match the comment in GridSearchCV ? The default is not warn in RandomizedSearchCV , or do you want the default to be warn in RandomizedSearchCV also?
If you are talking about ``'False'`` and ``False``, this was pushed by lesteve.

Reply:
I guess what @jnothman means here is that the docstring should be identical to the one in GridSearchCV. 

> The default is not warn in RandomizedSearchCV , or do you want the default to be warn in RandomizedSearchCV also?

I think the default is warn in RandomizedSearchCV (which is what we want) so the docstring for `return_train_score` of GridSearchCV and RandomizedSearchCV should be identical.

Reply:
> I think the default is warn in RandomizedSearchCV (which is what we want)

Ah sorry I misread the diff. I still think that we want the RandomizedSearchCV and GridSearchCV to have the same behaviour, i.e. `return_train_score='warn'`.